### PR TITLE
Feral Bikers

### PR DESCRIPTION
**Pull request by GrahammerTek**
#### Summary
Content "Adds a new monster; Feral biker"

#### Purpose of change
Adds feral bikers who are unarmored and quite rare, but carry shotguns and can spawn in groups up to three. This creates gun wielding enemies that use numbers over armor and equipment.

#### Describe the solution
Feral bikers spawn in later stages of the game. They have no armor or real melee weapons, so it can be best to run and let them waste their ammo, and then kill all of them in close quarters.

#### Describe alternatives you've considered
We could have them spawn mostly in rural areas.

#### Testing
Works fine. I'd like to know if other people think their spawn rates are too low/high.

### CODE REVIEW

**Comment by ISuckM8** [comment]
Maybe add shotgun shells to its loot table?

**Comment by GrahammerTek** [comment]
Right now they have a chance to drop shells in their gun, but it's all or nothing. I know a way to change this, but it would make it inconsistent with the other feral drops. I suppose this is different since it's a shotgun, so a few loose rounds shouldn't hurt.

File: data/json/monsterdrops/feral_humans.json [hunk 1]
```diff
@@ -532,5 +544,40 @@
...
+      { "group": "clothing_outdoor_pants", "damage": [ 1, 4 ] },
+      { "item": "sunglasses", "prob": 20, "damage": [ 1, 4 ] },
+      { "group": "underwear", "damage": [ 1, 4 ] },
+      { "item": "gloves_leather", "prob": 80, "damage": [ 1, 4 ] },
+      { "group": "salty_snacks", "count": [ 1, 3 ], "prob": 50 },
+      { "group": "snacks", "count": [ 1, 3 ], "prob": 60 },
+      { "item": "meth", "prob": 20, "charges": [ 1, 6 ] },
+      { "group": "meth_ingredients", "prob": 5 }
+    ]
   }
```
Comment (Night-Pryanik) [inline data/json/monsterdrops/feral_humans.json:582]: Socks needed.

Comment (Night-Pryanik) [inline data/json/monsterdrops/feral_humans.json:582]: Socks needed.

**Pull request merged**